pkg/skaffold/errors/errors.go

```diff
@@ -75,6 +76,21 @@ func ActionableErr(phase Phase, err error) *proto.ActionableErr {
 	}
 }
 
+// ActionableErr returns an actionable error message with suggestions
+func ActionableErrV2(phase Phase, err error) *protoV2.ActionableErr {
+	errCode, suggestions := getErrorCodeFromError(phase, err)
+	suggestionsV2 := make([]*protoV2.Suggestion, len(suggestions))
```

Comment:
do these need to be pointers? can we pass them by value instead

Reply:
Do you think that's a change we should make for v2? I mostly just did pointers since that's what the existing type uses